src/routers/secure/items.py

```
@@ -1066,3 +1043,28 @@ def mutation(i: MediaItem, s: Session):
             )
+
+
+# MediaMetadata return
+@router.get(
+    "/{item_id}/metadata",
+    summary="Get Media Item Metadata",
+    description="Get metadata for a media item using item ID",
+    operation_id="get_item_metadata",
+)
+async def get_item_metadata(
+    _: Request, item_id: int, db: Session = Depends(get_db)
```

Ruff (0.14.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)